lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/AccessRequirementUtils.java

```diff
@@ -33,12 +33,23 @@
 
 public class AccessRequirementUtils {
 	private static final UnmodifiableXStream X_STREAM = UnmodifiableXStream.builder().allowTypes(AccessRequirement.class).build();
-	private static HashMap<ObjectType, HashSet<ACCESS_TYPE>> NOT_ALLOWED_PERMISSIONS = new HashMap<>() {{
-		put(ObjectType.EVALUATION, new HashSet<>(Arrays.asList(ACCESS_TYPE.EXEMPTION_ELIGIBLE)));
-		put(ObjectType.TEAM, new HashSet<>(Arrays.asList(ACCESS_TYPE.EXEMPTION_ELIGIBLE)));
-		put(ObjectType.FORM_GROUP, new HashSet<>(Arrays.asList(ACCESS_TYPE.EXEMPTION_ELIGIBLE)));
-		put(ObjectType.ORGANIZATION, new HashSet<>(Arrays.asList(ACCESS_TYPE.EXEMPTION_ELIGIBLE)));
-		put(ObjectType.ENTITY, new HashSet<>(Arrays.asList(ACCESS_TYPE.EXEMPTION_ELIGIBLE)));
+	public static HashMap<ObjectType, HashSet<ACCESS_TYPE>> ALLOWED_ACCESS_TYPES = new HashMap<>() {{
```

Comment:
since you are directly exposing the map as a constant it should be immutable and the type should be map:
Map<ObjectType, Set<ACCESS_TYPE>> ALLOWED_ACCESS_TYPES = Map.of(...)
Same with the sets (note: there is no need create a list here):
put(ObjectType.ENTITY, Set.of(ACCESS_TYPE.CREATE, ACCESS_TYPE.DOWNLOAD, ACCESS_TYPE.READ..)